sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -23,6 +23,7 @@
     // Mesos may call registered() multiple times in the lifespan of a Scheduler process, specifically when there's
     // master re-election. Avoid performing initialization multiple times, which would cause resourcesQueue to be stuck.
     private final AtomicBoolean isAlreadyRegistered = new AtomicBoolean(false);
+    protected final OfferQueue offerQueue;
```

Comment:
Should offers be getting queued in `UninstallScheduler`? From its perspective, offerRescinded will attempt to remove offers that aren't being added. Would it make sense for `UninstallScheduler` to take advantage of this as well?